contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,133 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+    getPathsForAllAnswers() {
+        const allAnswerPaths = [];
+
+        // Log ids of all answers in section
```

Comment:
Super minor but I'd probably comment with "store" instead of "log" as it makes me expect a logging command
or even just `all answers in section` 
same in other places

Reply:
Ah good shout, thanks